src/app/create-room/CreateRoom.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+
+import { useMutation, useQuery } from '@apollo/client';
+
+import { Button, Spinner } from '../../packages';
+import {
+  CREATE_ROOM,
+  CreatRoomQuery,
+  GetRangeParticipantsQuery,
+  GET_RANGE_PARTICIPANTS,
+} from '../query';
+import { Styled } from './styled';
+
+export const CreateRoom = () => {
+  const [createRoom, { data: roomData, loading: roomLoading }] =
+    useMutation<CreatRoomQuery>(CREATE_ROOM);
+
+  const { data: rangeData, loading: rangeLoading } =
+    useQuery<GetRangeParticipantsQuery>(GET_RANGE_PARTICIPANTS);
+
+  const [maxParticipants, setMaxParticipants] = useState<null | number>(null);
+
+  const onChangeRange: React.FormEventHandler<HTMLInputElement> = (event) => {
+    setMaxParticipants(Number(event.currentTarget.value));
```

Comment:
Why do we need that Number cast? What values could be in currentTarget.value except of numbes?

Reply:
Input values is always string
But i remade with + convert